en/orm/saving-data.rst

```diff
@@ -874,8 +874,8 @@ the singular, underscored version of the association name. For example::
 Saving HasOne Associations
 --------------------------
 
-When saving hasOne associations, the ORM expects a single nested entity at the
-singular, underscored version of the association name. For example::
+When saving hasOne associations, the ORM expects a single nested entity named with the
+singular, lowercase version of the association name. For example::
```

Comment:
But its not just the the lowercase. For multi word names it is also the lowercase_underscore version.